src/main/java/com/jalasoft/webservice/controller/AudioController.java

```diff
@@ -82,10 +95,11 @@ public Response convert (@RequestParam("file") MultipartFile file,
             cri.setSamplingRate(samplingRate);
             cri.setFormat(format);
         } else {
+            logger.error("The cheksum send is not match - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
```

Comment:
review if this line do not super the 100 characters

Reply:
Done